### PR TITLE
Question activation on creation

### PR DESCRIPTION
**Pull request by ahmedqarmout2**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -461,6 +480,24 @@ app.post('/fetchqlist', function(req, res) {
...
+
+        req.session.adminQuestionList = null;
+        var html = questionList({
+            questions: questionsList
+        });
+
+        return res.status(200).send(html);
+    });
+
+    /*
```
Comment (Mohamed-A-Mohamed) [inline server.js:500]: Should have explain of why you commented out this code. Do you plan on using it later? Has it been replaced and will be deleted after?

File: server.js [hunk 2]
```diff
@@ -368,20 +368,39 @@ app.get('/questionlist', function(req, res) {
...
+
+        req.session.adminQuestionList = null;
+        var html = questionTable({
+            questions: questionsList
+        });
+
+        return res.status(200).send(html);
+    });
+
+    /* If this is the first time accessing it, fetch list from database.
```
Comment (Mohamed-A-Mohamed) [inline server.js:386]: Explanation of why this was commented would be nice just so when other people are working on this file they know why.

File: server.js (hunk 1 above)
Comment (ahmedqarmout2) [reply server.js:500]: oooh my bad. I am trying to replace it. To make sorting easier for future addition

File: server.js (hunk 2 above)
Comment (ahmedqarmout2) [reply server.js:386]: removing this route soon. Just keeping it now for testing

**Pull request merged**